server/src/main/java/com/cloud/server/ManagementServerImpl.java

```diff
@@ -4470,6 +4471,9 @@ public Map<String, Object> listCapabilities(final ListCapabilitiesCmd cmd) {
             regionSecondaryEnabled = true;
         }
 
+        final Integer fsVmMinCpu = Integer.parseInt(_configDao.getValue("storagefsvm.min.cpu.count"));
+        final Integer fsVmMinRam = Integer.parseInt(_configDao.getValue("storagefsvm.min.ram.size"));
```

Comment:
Not possible to use configkeys?

Reply:
StorageFsVmFileShareProvider class in plugins is not directly accessible from here. We can put the settings in FileShareProvider but that won't be totally accurate as these settings are only pertaining to storagefsvm provider.